tests/foreman/api/test_user.py

```diff
@@ -399,6 +399,37 @@ def test_negative_create_with_blank_authorized_by(self):
         with pytest.raises(HTTPError):
             entities.User(auth_source='').create()
 
+    @pytest.mark.tier1
+    def test_positive_table_preferences(self, module_target_sat):
+        """Create a user, create their Table Preferences, read it
+
+        :id: 10fda3f1-4fee-461b-a413-a4d1fa098a94
+
+        :expectedresults: Table Preferences can be accessed
+
+        :CaseImportance: Medium
+
+        :customerscenario: true
+
+        :BZ: 1757394
+        """
+        existing_roles = entities.Role().search()
+        password = gen_string('alpha')
+        user = entities.User(role=existing_roles, password=password).create()
+        name = "hosts"
+        columns = ["power_status", "name", "comment"]
+        sc = ServerConfig(auth=(user.login, password), url=module_target_sat.url, verify=False)
+        entities.TablePreferences(sc, user=user, name=name, columns=columns).create()
+        table_preferences = entities.TablePreferences(sc, user=user).search()
+        assert len(table_preferences) == 1
+        tp = table_preferences[0]
+        assert hasattr(tp, 'name')
+        assert hasattr(tp, 'columns')
+        assert tp.name == 'hosts'
```

Comment:
Non blocking :  Feels a bit redundant assertion , as we are setting the same variable above `name = "hosts"`
```suggestion
```

Reply:
We set some local variable in Python above, yes... but this is about asserting that the API indeed returned it!